src/open_inwoner/scss/components/UserFeed/UserFeed.scss

```diff
@@ -0,0 +1,93 @@
+.userfeed {
+  position: relative;
+
+  &__summery {
+    background-color: #fff;
+    color: var(--color-red-notification);
+    display: block;
+    line-height: var(--font-line-height-body);
+    position: relative;
+    overflow: hidden;
+    z-index: 1002;
```

Comment:
That's very specific. What's the rationale for this value (no objection, just curious)?

Reply:
You mean the Z-index? That's indeed my mistake 😞 - the other stuff is needed to prepare positioning for the pseudo-element (= the pipe symbol between list-items).